mmv1/third_party/terraform/resources/resource_compute_disk_async_replication.go.erb

```diff
@@ -0,0 +1,317 @@
+<% autogen_exception -%>
+package google
+<% if version != "ga" -%>
+
+import (
+	"fmt"
+	"regexp"
+	"time"
+	"log"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+<% if version == "ga" -%>
+	"google.golang.org/api/compute/v1"
+<% else -%>
+	compute "google.golang.org/api/compute/v0.beta"
+<% end -%>
+)
+
+func ResourceComputeDiskAsyncReplication() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDiskAsyncReplicationCreate,
+		Read:   resourceDiskAsyncReplicationRead,
+		Update: resourceDiskAsyncReplicationUpdate,
```

Comment:
If this resource cannot be updated we should omit an update function

Reply:
Omitted.